### PR TITLE
Fix .html and .css design

### PR DESCRIPTION
**Pull request by Tshepo45**
fix README file typos
Modify Package.json file

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Tshepo45,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

❗️❗️ Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project." The only optional section is the FAQ. Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌 

❗️❗️ Kindly fix your Pull Request title as requested [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), it should have a descriptive title telling you what changes are inside. Also please fix the Pull Request description section, it should have a short summary describing the introduced changes. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by NahnahAJ** [comment]
Hi @Tshepo45 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Please implement the changes the previous reviewer suggested you make :+1: 

1. You did job adding the README template. However, you left out some crucial parts such as 
- The Key Features Section
- The Future Features Section and 
- Lastly, the link to your MIT LICENSE file is broken. Please do well to fix all these so that the next reviewer can review your project :wink: 
2. Your PR Title and Description have still not been fixed please refer to his suggestions and fix these as well.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @Tshepo45 ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Please make sure all the files related to the PR are included in it, I am unable to see your `index.html` file so therefore not able to add inline comments to it. 😢 
- [ ] The container of the section should a header tag
- [ ] The logo should have a link (`a` tag).
- [ ] The hamburger button should have a `nav` tag.
  - The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices
- [ ] The container of the section should be a `section` tag.
  -  It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info
- [ ] Make sure your design matches the Figma 🖼️, your toolbar lacks some padding. See below 👇🏽 
  - Your design:
Screenshot 2023-01-14 at 00 54 25
  - Figma Design:
   Screenshot 2023-01-14 at 00 55 28

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 </head>
 <body>
 
+
   <section>
     <div class="header">
```
Comment (joskalenda) [inline index.html:14]: - [ ] As the previous reviewer said, consider changing this `div` on line 14 into a `header` tag, as per the requirement, typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation.



- [ ] [ OPTIONAL ] This should not be an image, despite that this comment is marked as optional, I would suggest you change this into a text for more flexibility on the design



- [ ] As mentioned by the previous reviewer, you have to make a few touches on the design to match the original one, take a look at these images below and make the necessary changes, make sure your page fits the mobile screen size

### Your design ⬇️ 



### Figma design ⬇️

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Tshepo45, |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Green linters
- [ ]... ✅ Good readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -9,15 +9,17 @@
 </head>
 <body>
 
+
   <section>
```
Comment (Nyame-Wolf) [inline index.html:13]: [OPTIONAL] I suggest you remove the section tag here. This being the navigation, using the `section` tag and `header` tag is redundant. The `header` alone would suffice.

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @Tshepo45  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Made changes as requested
- Linters are passing
- Correct github flow used.

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**